keyboards/exclusive/e6_rgb/keymaps/allleds/rules.mk

```diff
@@ -0,0 +1,34 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
```

Comment:
Lines 2 thru 32 and 34 share the same settings as the base rules.mk file you don't need to re-iterate them. 

all you really need in this file is

```
RGB_MATRIX_ENABLE = IS31FL3733
```

Reply:
remove the redundant lines